core/bump_objects.ts

```diff
@@ -166,7 +165,7 @@ function extractObjectFromEvent(
     case eventUtils.COMMENT_MOVE:
       object = workspace.getCommentById(
         (e as CommentCreate | CommentMove).commentId!,
-      ) as WorkspaceCommentSvg | null;
+      ) as AnyDuringMigration as RenderedWorkspaceComment;
```

Comment:
is the AnyDuringMigration cast necessary?

Reply:
In this PR yes, but it gets removed in the subsequent one.